README.rst

```diff
@@ -244,10 +244,15 @@ Compatibility with GMT/Python/NumPy versions
       - NumPy
     * - `Dev <https://github.com/GenericMappingTools/pygmt/milestones>`_ (upcoming release)
       - `Dev Documentation <https://www.pygmt.org/dev>`_ (reflects `main branch <https://github.com/GenericMappingTools/pygmt>`_)
-      - >=6.3.0
+      - >=6.4.0
+      - >=3.8
+      - >=1.20
+    * - `v0.8.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.8.0>`_ (latest release)
+      - `v0.8.0 Documentation <https://www.pygmt.org/v0.8.0>`_
+      - >=6.4.0
```

Comment:
Still semi-supporting GMT 6.3.0 as per https://github.com/GenericMappingTools/pygmt/issues/1991
```suggestion
      - >=6.3.0
```

Reply:
Thanks @weiji14 for pointing out this issue!